From 03745bc118a881bc1a5a4923cd062200e45e93c9 Mon Sep 17 00:00:00 2001
From: Troy Kisky <troy.kisky@boundarydevices.com>
Date: Mon, 16 Dec 2013 18:12:51 -0700
Subject: [PATCH 125/199] ARM: dts: imx6qdl-sabrelite: move pcie to
 imx6qdl-sabrelite.dtsi

PCIe is common to Quad, Dual, Dual Lite, and Solo variants
of the processor. So, move enabling from imx6q-sabrelite.dts
to imx6qdl-sabrelite.dtsi.

Signed-off-by: Troy Kisky <troy.kisky@boundarydevices.com>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx6q-sabrelite.dts    | 4 ----
 arch/arm/boot/dts/imx6qdl-sabrelite.dtsi | 4 ++++
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6q-sabrelite.dts b/arch/arm/boot/dts/imx6q-sabrelite.dts
index 8128fb9..96e4688 100644
--- a/arch/arm/boot/dts/imx6q-sabrelite.dts
+++ b/arch/arm/boot/dts/imx6q-sabrelite.dts
@@ -19,10 +19,6 @@
 	compatible = "fsl,imx6q-sabrelite", "fsl,imx6q";
 };
 
-&pcie {
-	status = "okay";
-};
-
 &sata {
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/imx6qdl-sabrelite.dtsi b/arch/arm/boot/dts/imx6qdl-sabrelite.dtsi
index c4aa504..5394e02 100644
--- a/arch/arm/boot/dts/imx6qdl-sabrelite.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabrelite.dtsi
@@ -187,6 +187,10 @@
 	};
 };
 
+&pcie {
+	status = "okay";
+};
+
 &ssi1 {
 	fsl,mode = "i2s-slave";
 	status = "okay";
-- 
1.8.5.3

